internal/integration/base/k8s.go

```diff
@@ -266,6 +268,10 @@ func (p *pod) Create(ctx context.Context, waitTimeout time.Duration) error {
 	return p.suite.WaitForPodToBeRunning(ctx, waitTimeout, p.namespace, p.name)
 }
 
+func (p *pod) SetQuiet(quiet bool) {
```

Comment:
This is to silence stderr output from tests like this, which expect commands to fail